dev/README_RELEASE_AIRFLOW.md

```diff
@@ -205,6 +205,7 @@ The Release Candidate artifacts we vote upon should be the exact ones we vote ag
 - Add a commit that updates `CHANGELOG.md` to add changes from previous version if it has not already added.
 For now this is done manually, example run  `git log --oneline v2-2-test..HEAD --pretty='format:- %s'` and categorize them.
 - Add section for the release in `UPDATING.md`. If no new entries exist, put "No breaking changes" (e.g. `2.1.4`).
+- Update the `REVISION_HEADS_MAP` at airflow/utils/db.py to include the revision head of the release even if there are no migrations.
```

Comment:
We should have a pre-commit to verify that a tuple containing the current Airflow version is added to `REVISION_HEADS_MAP` to avoid forgetting it.

Reply:
It looks like a pre-commit won't be possible in this case as we only update the migration head when there's a new airflow release.

Reply:
How about something as simple as:

```python
from airflow.utils.db import REVISION_HEADS_MAP
from airflow.version import version as airflow_version
from packaging.version import Version


if not Version(airflow_version).is_prerelease and airflow_version not in REVISION_HEADS_MAP:
    raise Exception(f"Airflow version {airflow_version} is not in the revision map {REVISION_HEADS_MAP}")
```

Reply:
i was also thinking about this issue.  i saw that we had a table in the docs which contains this mapping.  so what i was thinking to do was store that table in yaml form, and it could be used to generate the docs table and to support CLI commands which need to map version to revision, and that's what i do in https://github.com/apache/airflow/pull/21601

Reply:
Looks good, I think the challenge would be to get the migration heads of the versions and discard every other migration in-between when going from version to version. It's still necessary to have a pre-commit or a ci job that makes sure we have the migration head updated before a release